az-*: update links and page description

sebastiaanspeck opened the pull request:
<!--
Thank you for contributing!
Please fill in the following checklist, removing items that do not apply.
See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
-->

- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

gutjuri commented on pages.de/common/az-config.md:
```diff
@@ -1,8 +1,8 @@
 > Verwalten der Azure CLI-Konfiguration.
-> Teil von `azure-cli`.
-> Weitere Informationen: <https://learn.microsoft.com/cli/azure/config>.
+> Teil von `az`, der Befehlszeilenschnittstelle von Azure.
```
```suggestion
> Teil von `az`, der Kommandozeilenschnittstelle von Azure.
```

gutjuri commented on pages.de/common/az-logout.md:
```diff
@@ -1,8 +1,8 @@
 > Von Azure-Abonnements abmelden.
-> Teil von `azure-cli`.
-> Weitere Informationen: <https://learn.microsoft.com/cli/azure/reference-index#az_logout>.
+> Teil von `az`, der Befehlszeilenschnittstelle von Azure.
```
```suggestion
> Teil von `az`, der Kommandozeilenschnittstelle von Azure.
```

gutjuri reviewed (changes requested):
Thanks you! Looks fine to me, apart from the german word "Befehlszeile", which I changed to "Kommandozeile" in my suggestion.

sebastiaanspeck commented:
> Thanks you! Looks fine to me, apart from the german word "Befehlszeile", which I changed to "Kommandozeile" in my suggestion.

Changed in all three az-files. Thank you for the suggestion!

gutjuri commented:
LGTM

kant commented on pages.es/common/az-appconfig.md:
```diff
@@ -2,7 +2,7 @@
 
 > Administra configuraciones de aplicaciones en Azure.
```
Administra configuraciones --> Administra las configuraciones

kant commented on pages.es/common/az-bicep.md:
Line 11: Crear un archivo --> Crea un archivo

kant commented on pages.es/common/az-feedback.md:
```diff
@@ -1,8 +1,8 @@
+> Parte de `az`, el cliente de línea de comandos para Microsoft Azure.
+> Más información: <https://learn.microsoft.com/es-es/cli/azure/reference-index#az_feedback>.
 
 - Envia comentarios al equipo de Azure CLI:
```
Envia --> Envía

acuteenvy reviewed (commented):
There's recently been [a PR](https://github.com/tldr-pages/tldr/pull/10929) removing the "command-line client for Microsoft Azure" part as per the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#descriptions).

kbdharun reviewed (changes requested):
Thanks for your contribution. As @acuteenvy says, we can drop the `the command-line client for Microsoft Azure` part. But the `azure-cli` -> `az` change LGTM as we have the page named `az` too (I will add an alias page named `azure-cli`).

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/tldr-pages/tldr?pullRequest=10963) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/tldr-pages/tldr?pullRequest=10963) before we can accept your contribution.<br/>**14** out of **15** committers have signed the CLA.<br/><br/>:white_check_mark: sebastiaanspeck<br/>:white_check_mark: dadav<br/>:white_check_mark: acuteenvy<br/>:white_check_mark: IMHOJEONG<br/>:white_check_mark: TheBulaDev<br/>:white_check_mark: kbdharun<br/>:white_check_mark: reinhart1010<br/>:white_check_mark: Vishnudas2003<br/>:white_check_mark: aditya955<br/>:white_check_mark: jnmcfly<br/>:white_check_mark: cmhpedro<br/>:white_check_mark: npv12<br/>:white_check_mark: ckorobov<br/>:white_check_mark: dmytrovoytko<br/>:x: Sebastiaan Speck<br/><hr/>**Sebastiaan Speck** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/tldr-pages/tldr?pullRequest=10963) it.</sub>

kbdharun reviewed (changes requested):
We don't specify the locale in Microsoft links as it will automatically redirect to the reader's preferred locale (it is especially useful for Bilingual speakers).

sebastiaanspeck commented:
Want to edit the new az-pages as well.
